content/docs/setup/kubernetes/spec-requirements/index.md

```diff
@@ -38,3 +38,47 @@ cluster must satisfy the following requirements:
 
 * _**`NET_ADMIN` capability**:_ If pod security policies are enforced in your cluster and unless you use the [Istio CNI Plugin](/docs/setup/kubernetes/istio-cni-install/), your pods must have the `NET_ADMIN` capability allowed.
 See [Required Pod Capabilities](/help/ops/setup/required-pod-capabilities/).
+
+* _**Service Ports and Protocols**_: Ensure that your service's ports and protocols are available and don't conflict with the [Istio used ports and protocols](#istio-used-ports) for the Istio components be used.
+
```

Comment:
Please remove this sentence. There is no conflict with any Istio ports on Kubernetes. ever.

Reply:
Done